src/components/CategoryMenu.jsx

```diff
@@ -1,109 +1,113 @@
-import React, { useState } from 'react'
-import { useDispatch, useSelector } from 'react-redux'
-import { Link } from 'react-router-dom'
-import { Menu, Segment } from 'semantic-ui-react'
-import { getArticles } from '../modules/getArticles'
-import { useTranslation } from 'react-i18next'
-import RegistrationForm from './RegistrationForm'
-import SubscriptionForm from './SubscriptionForm'
+import React, { useState } from "react"
```

Comment:
Please sort out your formatters so you all have either single or double quotes!

Reply:
Yes!